cumulus/parachains/runtimes/bridge-hubs/common/Cargo.toml

```diff
@@ -12,6 +12,7 @@ scale-info = { features = ["derive"], workspace = true }
 frame-support = { workspace = true }
 sp-core = { workspace = true }
 sp-runtime = { workspace = true }
+sp-std = { workspace = true }
```

Comment:
We shouldn't keep on using `sp-std`: https://github.com/paritytech/polkadot-sdk/issues/2101